ledger-service/http-json/src/main/scala/com/digitalasset/http/Endpoints.scala

```diff
@@ -91,11 +91,23 @@ class Endpoints(
         (implicit lc => httpResponse(allocateParty(req)))
       case req @ HttpRequest(GET, Uri.Path("/v1/packages"), _, _, _) =>
         (implicit lc => httpResponse(listPackages(req)))
-      // format: off
-      case req @ HttpRequest(GET,
-          Uri(_, _, Slash(Segment("v1", Slash(Segment("packages", Slash(Segment(packageId, Empty)))))), _, _),
-          _, _, _) => (implicit lc => downloadPackage(req, packageId))
-      // format: on
+
+      case req @ HttpRequest(
+            GET,
+            Uri(
+              _,
+              _,
+              Slash(Segment("v1", Slash(Segment("packages", Slash(Segment(packageId, Empty)))))),
+              _,
+              _,
+            ),
+            _,
+            _,
+            _,
+          ) => (
+        implicit lc => downloadPackage(req, packageId)
+      )
```

Comment:
Unfortunately the formatter cannot handle this while keeping this readable. I would recommend to revert this and maybe add a comment to let readers know why.

Reply:
did revert that